controllers/grid/settings/category/form/CategoryForm.inc.php

```diff
@@ -292,6 +315,80 @@ function execute(...$functionArgs) {
 		parent::execute(...$functionArgs);
 		return $category;
 	}
-}
 
+	/**
+	 * Get a list of all subeditor IDs assigned to this category
+	 *
+	 * @param $categoryId int
+	 * @param $contextId int
+	 * @return array
+	 */
+	public function _getAssignedSubEditorIds($categoryId, $contextId) {
+		import('classes.core.Services');
```

Comment:
You shouldn't need this import statement. The services classes get loaded early on and should be globally available.